src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -525,4 +525,9 @@ To use the Log API directly, you'd edit the [Log API endpoint configuration](/do
 
 ### Trace API [#trace-api]
 
-FedRAMP is not yet available for the Trace API itself, but note that some of our agents report trace data and you can [enable FedRAMP for some of those agents](#agents). 
\ No newline at end of file
+Notes about FedRAMP compliancy for trace data: 
+* Trace data is reported by some of our agents, like our APM agents, browser agent, and mobile agent. To enable FedRAMP compliancy for that data, you would [enable FedRAMP for the applicable agent](#agents).
```

Comment:
I think "compliance" might be a better form here, unless "compliancy" is used elsewhere in FedRamp docs.